javascripts/buzz.js

```diff
@@ -119,11 +119,12 @@ app.buzz = {
               d.authorName = d.author;
             }
             d.updatedDate = jQuery.timeago(new Date(d.sys_created));
+            var dateOfBuzz = moment(d.sys_created).format('YYYY/DD/MM')
             d.sys_description = d.sys_description[0].substr(0,197) + '...';
-            if (d.sys_url_view[0].startsWith('http://developerblog.redhat.com')) {
+            if (d.sys_url_view[0].startsWith('http://developerblog.redhat.com') || d.sys_url_view[0].startsWith('https://developerblog.redhat.com') || d.sys_url_view[0].startsWith('http://developers.redhat.com/blog/') || d.sys_url_view[0].startsWith('https://developers.redhat.com/blog/')) {
               d.permanentLink = d.sys_url_view;
             } else {
-              d.permanentLink = "//planet.jboss.org/post/" + d.sys_content_id;
+              d.permanentLink = "//developers.redhat.com/blog/" + dateOfBuzz +"/" + d.sys_content_id;
```

Comment:
I'm not sure this 'dateOfBuzz' does what you expect. See the buzz page errors on: http://developers-pr.stage.redhat.com//pr/963/blinkr/671/report.html